tests/_fixtures/SetUpTearDownTest.php

```diff
@@ -0,0 +1,34 @@
+<?php
+/**
+ * Copyright (c) Enalean, 2018. All Rights Reserved.
+ *
+ * This file is a part of Tuleap.
+ *
+ * Tuleap is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * Tuleap is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with Tuleap. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+class SetUpTearDownTest
+{
+    public function setUp()
+    {
+        // initialize stuff
+        $this->stuff = 'foo';
+    }
+
+    public function tearDown()
+    {
+        // clean
+        $this->stuff = '';
+    }
+}
```

Comment:
Linter says newline is missing

Reply:
Which linter? Here I have a bunch of fixtures files that mimic legacy code which does not necessarily respect our latest standards. There is no need to pass those files to the linter.